java/gradlew.bat

```diff
@@ -1,103 +1,103 @@
-@rem
```

Comment:
Is this just line endings changes?

Reply:
I'm not sure if there are other differences between the 6.3 and 7.3 wrapper scripts. Since it's the Windows script, it makes sense to have the Windows line endings.